ios/MagnetScannerReact.swift

```diff
@@ -14,12 +17,41 @@ import Foundation
   }
   
   @objc func stop() {
-    scanner.stop()
+    // We don't stop scanning in iOS, we let the OS to
+    // throttle the notifications of beacons found.
+    // scanner.stop()
```

Comment:
I think this needs a little more explanation.

We don't stop scanning when the app goes into the background as we want bluetooth scanning to continue. If we stop scanning we have no way of starting again until the app comes to the foreground. When the app is in the background and the scanner is still running, the OS will periodically call the scan-callback with results.